content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md

```diff
@@ -16,207 +16,255 @@ aliases:
+RKVST authentication uses the industry-standard OIDC Client Credentials Flow.
 
 ## Creating a Custom Integration
 
-{{< warning >}}
-**Warning:** You may only create and manage App Registrations with an **Administrator** level account.
-{{< /warning >}}
-
-When enabling non-interactive access to RKVST, you ***must*** create your first Integration in the **RKVST UI**.
-
-### Using the RKVST UI to create an Integration (First-Time Setup)
-
-1. As an Administrator, open the `Settings` interface.
-
-{{< img src="Settings.png" alt="Rectangle" caption="<em>Settings</em>" class="border-0" >}}
-
-2. Navigate to the `Integrations` tab.
-
-{{< img src="IntegrationsTab.png" alt="Rectangle" caption="<em>Navigate to Integrations</em>" class="border-0" >}}
-
-3. Click the `Custom` box and the following form should appear:
-
-{{< img src="CreateCustomForm.png" alt="Rectangle" caption="<em>Custom Integration Webform</em>" class="border-0" >}}
-
-3. Enter any display name you'd like, then click `Confirm`.
-
-{{< note >}}
- You can optionally add any Custom Claims at this step. You must ensure they do not start with `jit_` or use any of the [well-known reserved claims](https://auth0.com/docs/security/tokens/json-web-tokens/json-web-token-claims#reserved-claims). The Custom Claims can be used in an [Attribute-Based Access Control (ABAC) policy](/platform/administration/managing-access-to-an-asset-with-abac) to grant permissions. 
-{{< /note >}}
-
-{{< img src="Confirm.png" alt="Rectangle" caption="<em>Completed Web Registration</em>" class="border-0" >}}
-
-4.  You will then be presented with the `CLIENT_ID` and `SECRET` required by the archivist token endpoint.
-
-{{< caution >}}
-**Caution:** You **must** take note of the `SECRET` at this point - it can **not** be viewed again later and you will have to generate a new one.
-{{< /caution >}}
-
-{{< img src="RecordClientIDandSecret.png" alt="Rectangle" caption="<em>Record your Client ID and Secret</em>" class="border-0" >}}
-
-5. Now that you have created your Custom Integration, follow the steps below to [test generating a token](./#getting-a-token-with-your-app-registration) and [ensure you can access the RKVST API](./#testing-your-access).
-
 {{< note >}}
-**Note:** By default, newly created Integrations will always have a Non-Administrator permission to the API, you must add the Integration as an Administrator to elevate it's permissions.
-
-You can add a Custom Integration as an Administrator using the `Settings` screen, where the issuer will be `https://app.rkvst.io/appidpv1` and the subject will be your Integration's `CLIENT_ID`.
+**Note:** Creating App Registrations requires **Administrator** privileges.  
+If `Settings` does not appear in the navigation, see your RKVST Administrator for access.
 {{< /note >}}
 
-### Using the App Registrations API to create an Integration
+### Using the RKVST App to Create an Integration (First-Time Setup)
+
```

Comment:
Is there a reason why these bullet points are all 1. ? I still see a numbered sequence in the browser.

Reply:
I've just found the reason while reviewing the style guide changes :-)